modules/admission-controller-about.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-controllers.adoc
+
+[id="admission-controller-about_{context}"]
+= About admission controllers
+
+Admission controllers are used to help regulate how {product-title} {product-version} functions.  Admission control plug-ins intercept requests to the master API to validate resource requests and ensure policies are adhered to. For example, admission control plug-ins are commonly used to enforce security policy, resource limitations or configuration requirements.
+
+Admission control plug-ins run in an admission chain, starting with mutating controllers that can modify resources, followed by validating controllers. Some controllers can both modify and validate. If any controller in the sequence rejects a request, the whole chain is aborted and an error is returned. Mutating controllers run prior to validating controllers so that modifications that they trigger can also be validated.
```

Comment:
avoid the term "controller" for admission plugins. Controller is highly misleading.

Reply:
Thank you Stefan for your review. Following on from our Slack conversation, I have updated all references throughout the document, titles and file names to be more consistent here. Namely, I have used the term admission plug-in throughout.